modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/AccountResourceTest.java

```diff
@@ -173,6 +174,53 @@ public void testDeleteOrganizationAccountsByExternalReferenceCode()
 					organization.getOrganizationId()));
 	}
 
+	@Override
+	@Test
+	public void testDeleteOrganizationByExternalReferenceCodeAccountsByExternalReferenceCode()
```

Comment:
why we cannot use the default test in the base class?

Reply:
@smottal it is not suited for complete testing, for instance this method created 3 accounts and the base one only one. Since the name implies ...AccountsByERC, we have to check for multiple ERC, I believe.